tests/test_bmidelta.py

```diff
@@ -724,7 +714,7 @@ def test_get_end_time(self, tmp_path):
         delta.initialize(p)
         assert delta.get_end_time() == pytest.approx(np.finfo('d').max)
 
-    @pytest.mark.xfail(raises=AttributeError, strict=True,
+    @pytest.mark.xfail(raises=AttributeError, strict=False,
```

Comment:
same here, why does this one not strict fail anymore?

Reply:
Yeah same as previous, not too sure why it works now, but I guess a lot has changed upstream in pyDeltaRCM since the BMI interface was put together.